File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p02922/p02922_num0_parsed.cpp
#include <iostream>

int main()
{
    int A, B;
    std::cin >> A >> B;

    int strips_needed = 0;

    // Calculate the number of strips needed to extend the socket
    // until we have B or more empty sockets
    while (A < B)
    {
        A += A - 1; // Extend one empty socket to A empty sockets
        strips_needed++;
    }

    std::cout << strips_needed << std::endl;

    return 0;
}

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p02922/p02922_num0_parsed.py
# Program to find the minimum number of power strips required to extend one socket into B or more empty sockets

# Get input values for A and B
A, B = map(int, input().split())

# Calculate the minimum number of power strips required
min_power_strips = (B-1) // (A-1) + 1

# Print the minimum number of power strips required
print(min_power_strips)